src/lib/felaSanitizeCssPlugin.ts

```diff
@@ -0,0 +1,54 @@
+const isValidCssValue = (value: any) => {
+  if (typeof value !== 'string') {
+    return true
+  }
+
+  const openingBrackets = '({['
+  const closingBrackets = ')}]'
+
+  const openingBracketsStack = []
+
+  for (let i = 0; i < value.length; ++i) {
```

Comment:
Could we get an inline comment here as to what this is trying to achieve?

Reply:
sure, will add this